plugins/linear-genome-view/src/LinearGenomeView/components/LinearGenomeView.test.js

```diff
@@ -33,9 +33,29 @@ describe('<LinearGenomeView />', () => {
     session.addAssemblyConf(assemblyConf)
     session.addView('LinearGenomeView', { id: 'lgv' })
     const model = session.views[0]
-    const { container, findByText } = render(<LinearGenomeView model={model} />)
-    await findByText('Open')
+    model.setWidth(800)
+    const {
+      container,
+      findByText,
+      findByTestId,
+      findByPlaceholderText,
+    } = render(<LinearGenomeView model={model} />)
+    await findByText('Select assembly to view')
+    await findByText('Enter a sequence or locstring')
+
+    const autocomplete = await findByTestId('autocomplete')
+    const inputBox = await findByPlaceholderText('Search for location')
+
     expect(container.firstChild).toMatchSnapshot()
+    expect(model.displayedRegions.length).toEqual(0)
+    autocomplete.focus()
+    fireEvent.change(inputBox, {
+      target: { value: 'ctgA:1..5' },
+    })
+    fireEvent.keyDown(autocomplete, { key: 'ArrowDown' })
+    fireEvent.keyDown(autocomplete, { key: 'ArrowDown' })
+    fireEvent.keyDown(autocomplete, { key: 'Enter', code: 'Enter' })
+    expect(model.displayedRegions.length).toEqual(1)
```

Comment:
nice work on the testing. why are arrowdown needed?

Reply:
I am testing the alternate search option from the dropdown.
https://github.com/GMOD/jbrowse-components/pull/1642/files#diff-82a164c4727a03a6f61491bcd006b102c68cd5f7ef74c19adfa73671fba37b61R87-R90 . I can change this to test the input value on enter if we do not want arrow down.